sample/README.adoc

```diff
@@ -24,6 +24,17 @@ $ mvn clean install -Pdistribution,sample
 The sample WAR file is not deployed by default. Just copy the generated WAR file as "hawkular-accounts-sample.war" into
 Wildfly's deployment directory from the "distribution" module.
 
+Note that for your own component, you'll have to adapt the standalone.xsl to include your deployment under the
+"secure-deployments" section.
+
+To test the sample, you can perform the following curl call:
+
+[source,bash]
+---
+$ curl -u jdoe:password -H "Accept: application/json" -H "Content-Type: application/json" -X POST -d '{"name":"test"}'
+http://localhost:8080/hawkular-accounts-sample/samples
```

Comment:
PR looks good to me.
For the future, as curl examples are "cheap" we can add more use cases GET/POST/PUT/DELETE to show a global round.
Whatever is just copy/paste from an user perspective helps to use/jump into the code more easily.
But that is not a stopper from my side, please go ahead with the merge.
Thanks,